src/styles/card_style.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: $card-width;
+  font-family: inherit;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $color-white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  transition: transform 300ms linear, color 300ms linear;
+
+  &__img {
+    display: block;
+    width: $img-width;
+    height: $img-height;
+    margin: 16px 0 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__block-price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
```

Comment:
Set margin always in one direction only top or bottom if you have some situation like this